experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -306,4 +327,63 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(
+    envWithoutDefaults: ENVIRONMENT
+  ): void {
+    const logExportersList = envWithoutDefaults.OTEL_LOGS_EXPORTER;
+    if (!logExportersList) {
+      diag.warn(`No log exporters specified. Logs will not be exported.`);
+      return;
+    }
+
+    const exporters: LogRecordExporter[] = [];
+    const enabledExporters = filterBlanksAndNulls(logExportersList.split(','));
+
+    if (enabledExporters.includes('none')) {
+      diag.warn(
```

Comment:
The user explicitly set `none` so this is expected behavior. 

```suggestion
      diag.info(
```